### PR TITLE
cruft 2.9.0 (new formula)

### PR DESCRIPTION
**Pull request by shmygol**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/cruft.rb [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+  resource "requests" do
+    url "https://files.pythonhosted.org/packages/e7/01/3569e0b535fb2e4a6c384bdbed00c55b9d78b5084e0fb7f4d0bf523d7670/requests-2.26.0.tar.gz"
+    sha256 "b8aa58f8cf793ffd8782d3d8cb19e66ef36f7aba4353eec859e74678b01b07a7"
+  end
+
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/71/39/171f1c67cd00715f190ba0b100d606d440a28c93c7714febeca8b79af85e/six-1.16.0.tar.gz"
+    sha256 "1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926"
+  end
```
Comment (carlocab) [inline Formula/cruft.rb:101]: Let's do
```ruby
depends_on "six"
```
instead.

Comment (shmygol) [reply Formula/cruft.rb]: Done

**Comment by shmygol** [comment]
@carlocab may the failure be solved by upgrading pip? https://cryptography.io/en/latest/installation/

**Comment by carlocab** [comment]
No, that's not it. It's trying to build `cryptography` from source, but it won't be able to do that without a Rust compiler.

It only needs `cryptography` to build `poetry` (which it uses to build `cruft`), so we can probably skip that step and just build `cruft` directly using Homebrew `poetry`. I'll push a commit to see if this works...

**Comment by carlocab** [comment]
Actually, something I'm confused about: the `poetry` build seems to want `cryptography`, but the Homebrew `poetry` formula does not declare a `cryptography` resource.

**Comment by shmygol** [comment]
Looks like `cryptography` is an indirect dependency. In poetry.lock of the project poetry I see `poetry` => `keyring` => `secretstorage` => `cryptography`

**Comment by carlocab** [comment]
Yes, the formula has `keyring` resource, but that doesn't seem to pull in `cryptography`. Weird.

**Review by carlocab — APPROVED** [review]
Thanks for your contribution!

**Pull request closed without merging**